packages/@react-types/view/src/content.d.ts

```diff
@@ -14,5 +14,8 @@ import {DOMProps, StyleProps} from '@react-types/shared';
 import {ReactNode} from 'react';
 
 export interface ContentProps extends DOMProps, StyleProps {
+  /**
+   * Content content
```

Comment:
This feels like a joke, but is following the other docs.

Reply:
easter egg it, so meta

Reply:
![content](https://i.imgflip.com/42hx7z.jpg)